R/dist_spec.R

```diff
@@ -190,43 +209,48 @@ c.dist_spec <- function(...) {
 #' # The mean of the sum of two distributions
 #' mean(dist1 + dist2)
 mean.dist_spec <- function(x, ..., ignore_uncertainty = FALSE) {
```

Comment:
L187 says "ignoring any uncertainty" but I see there's an option to not do that here so maybe the doc should be updated?

Reply:
Fixed in 6b35983fe3604b1980b2260aa8a8acce918f58bf